modules/migration-migrating-on-prem-to-cloud.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc
+// * migration_toolkit_for_containers/advanced-migration-options-mtc.adoc
+
+[id="migration-migrating-applications-on-prem-to-cloud_{context}"]
+= Migrating an application from on-premises to a cloud-based cluster
+
+You can migrate from a source cluster that is behind a firewall to a cloud-based destination cluster by establishing a network tunnel between the two clusters. The `crane tunnel-api` command establishes such a tunnel by creating a VPN tunnel on the source cluster and then connecting to a VPN server running on the destination cluster. The VPN server is exposed to the client using a load balancer address on the destination cluster.
+
+A service created on the destination cluster exposes the source cluster's API to {mtc-short}, which is running on the destination cluster.
+
+.Prerequisites
+
+* The system that creates the VPN tunnel must have access and be logged in to both clusters.
+* It must be possible to create a load balancer on the destination cluster. Refer to your cloud provider to ensure this is possible. (The only time Jason saw it's not possible on a single node OpenShift cluster)
+* Have names prepared to assign to namespaces, on both the source cluster and the destination cluster, in which to run the VPN tunnel. These namespaces should not be created in advance. For information on namespace rules, see \https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names.
+* When connecting multiple firewall-protected source clusters to the cloud cluster, each source cluster requires its own namespace.
+* OpenVPN server is installed on the destination cluster.
+* OpenVPN client is installed on the source cluster.
+* When configuring the source cluster in {mtc-short}, the API URL takes the form of https://proxied-cluster.${namespace}.svc.cluster.local.
+  See https://docs.openshift.com/container-platform/4.9/migration_toolkit_for_containers/advanced-migration-options-mtc.html#migration-migrating-applications-api_advanced-migration-options-mtc, step 3: _Create a MigCluster CR manifest for each remote cluster_ or...there's a web console way to do this to, but we can't find the documentation for it.
+* The `crane` utility is installed. (??? Need to add a topic on how to install it. ???)
```

Comment:
Jason posted the command to install it in one of the comments on your Jira.
![image](https://user-images.githubusercontent.com/31725355/153765953-c7db87d3-4c17-4448-9241-e03cfe70fdc6.png)

Reply:
Added this as a step in the procedure.